common/include/pcl/common/time.h

```diff
@@ -119,14 +113,9 @@ namespace pcl
       inline ScopeTime (const char* title) : 
         title_ (std::string (title))
       {
-        start_time_ = boost::posix_time::microsec_clock::local_time ();
       }
 
-      inline ScopeTime () :
-        title_ (std::string (""))
-      {
-        start_time_ = boost::posix_time::microsec_clock::local_time ();
-      }
+      inline ScopeTime () : ScopeTime ("") {};
```

Comment:
What about having a single constructor with default argument?

```cpp
ScopeTime (const std::string& title = "") : title_ (title) {}
```

Reply:
Didn't wanted to change the type from `const char* title` to `std::string` in this PR.